tiatoolbox/dataloader/wsireader.py

```diff
@@ -84,15 +85,148 @@ def slide_info(self):
         """
         raise NotImplementedError
 
-    def read_region(self, start_w, start_h, end_w, end_h, level=0):
+    def relative_level_scales(self, target_scale, units):
```

Comment:
Can we rename this to `get_relative_level_for_scales` ? This is getter function so make it more verbose is easier to read.

Reply:
I was going for something similar to `level_downsamples`, except here it's relative to a given resolution instead of baseline. I like the suggestion but I don't think the word order makes sense this way round. How about `find_relative_scale_for_levels`?